src/web/.ubiquity-os.config.yml

```
@@ -7,6 +7,9 @@ incentives:
   dataPurge: {}
   formattingEvaluator: {}
   permitGeneration: {}
+  reviewIncentivizer:
```

```suggestion
reviewIncentivizer: {}
```

Let's set the default like the other elements.